daml-lf/archive/src/test/scala/com/digitalasset/daml/lf/archive/DecodeV1Spec.scala

```diff
@@ -714,9 +714,9 @@ class DecodeV1Spec
 
     s"reject comparison builtins as is if version >= ${LV.Features.genComparison}" in {
 
-      forEveryVersionSuchThat(_ < LV.Features.genComparison) { version =>
+      forEveryVersionSuchThat(_ >= LV.Features.genComparison) { version =>
         val decoder = moduleDecoder(version)
-        forEvery(genericComparisonBuiltinCases) { (proto, _) =>
+        forEvery(comparisonBuiltinCases) { (proto, _) =>
```

Comment:
I’m confused, where did we rename that? Are we still using `genericComparisonBuiltinCases`?

Reply:
The test ` s"reject comparison builtins as is if version >= ${LV.Features.genComparison}" `  was just wrong. 
Its body was copied from `"translate generic comparison builtins as is if version < ${LV.Features.genComparison}"` but it was never adapted it. 
You can check the body of those tests were excatly the same before my fix.

Reply:
ah right, thanks for the explanaiton.